docs/stacks-guide.md

```diff
@@ -103,7 +105,7 @@ documentation][crossplane-install-docs].
 
 ### Create the application namespace
 
-[Kubernetes namespaces][kubernetes-namespace-docs] are used to isolate
+[Kubernetes namespaces][kubernetes-namespaces-docs] are used to isolate
 resources in the same cluster, and we'll use them in our Crossplane
 control cluster too. Let's create a namespace for our application's
 resources. We'll call it `app-project1-dev` for the purposes of this
```

Comment:
The part below where it says "we will be setting up some cloud provider
configuration in that namespace" is a bit confusing because we have created this namespace for application resources